src/TenantCloud/RentlerSDK/Fake/FakeOrganizationsApi.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace TenantCloud\RentlerSDK\Fake;
+
+use TenantCloud\RentlerSDK\Organizations\OrganizationsApi;
+use TenantCloud\RentlerSDK\Organizations\PaginatedOrganizationsResponseDTO;
+use TenantCloud\RentlerSDK\Organizations\SearchOrganizationsDTO;
+
+class FakeOrganizationsApi implements OrganizationsApi
+{
+	public const FAKE_ITEM = [
+		'organizationId' => 'string1',
```

Comment:
Change default faker to three organizations: 

```
TC-R
TC-CP
TC-TC
```

and set `partnerId` to `tc` for all